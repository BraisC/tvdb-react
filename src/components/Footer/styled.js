import styled from 'styled-components';

const Footer = styled.footer`
  grid-area: footer;
  background: var(--color-primary);
  transition: 0.2s ease;
`;

export const Styled = {
  Footer,
};
