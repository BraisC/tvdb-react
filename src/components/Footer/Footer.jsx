import React from 'react';
import { logo } from 'images';
import { Styled } from './styled';

const Footer = () => (
  <Styled.Footer>
    <Styled.Credits>Made by Brais Cao</Styled.Credits>
    <Styled.Attribution>
      <Styled.Credits>Powered by</Styled.Credits>
      <Styled.Logo src={logo} alt="TMDB Logo" />
    </Styled.Attribution>
  </Styled.Footer>
);

export default Footer;
