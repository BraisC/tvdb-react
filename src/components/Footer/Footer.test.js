import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('<Footer/>', () => {
  test('if Footer component renders', () => {
    expect(shallow(<Footer />)).toMatchSnapshot();
  });
});
