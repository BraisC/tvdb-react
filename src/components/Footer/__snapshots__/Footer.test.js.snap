// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> if Footer component renders 1`] = `ShallowWrapper {}`;
